testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -4928,6 +4928,37 @@ private static void checkIf(SqlOperatorFixture f) {
     f.checkNull("regexp_contains(cast(null as varchar), cast(null as varchar))");
   }
 
+  @Test void testRegexpFunc() {
+    final SqlOperatorFixture f = fixture().setFor(SqlLibraryOperators.REGEXP);
```

Comment:
fixture().setFor(SqlLibraryOperators.REGEXP).withLibrary(SqlLibrary.Spark) ？

Reply:
`withLibrary` is optional.
`f0.forEachLibrary(list(functionAlias.libraries), consumer)` will automatically load the corresponding Library type.